osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -18,7 +18,6 @@ protected override void InitialiseDefaults()
             Set(OsuConfig.MouseSpeed, 1.0);
 
             Set(OsuConfig.Username, string.Empty);
-            Set(OsuConfig.Password, string.Empty);

```

Comment:
i'm not sure you want to remove the default init. that means you need to handle a null situation, right? i'd rather stick to `string.Empty` everywhere just so we don't have ambiguity of null vs empty string.